torchinductor/lowering.py

```diff
@@ -249,9 +249,17 @@ def promote_constants(inputs):
     ]
 
 
-def make_pointwise(fn, override_dtype=None, override_device=None, override_bool=None):
-    def inner(*inputs: List[TensorBox]):
+def make_pointwise(
+    fn, override_dtype=None, override_device=None, override_bool=None, allow_alpha=False
+):
+    def inner(*inputs: List[TensorBox], alpha=None):
         inputs = promote_constants(inputs)
+        if allow_alpha:
+            if alpha is not None and alpha != 1:
+                inputs = list(inputs)
+                inputs[-1] = mul(inputs[-1], alpha)
```

Comment:
assert alpha is a scalar?

Reply:
`mul()` already has asserts on its inputs.

Reply:
Hmm, I assumed mul would be able to take tensors, where alpha was required to be a scalar. As long as mul asserts, sgtm!

Reply:
Often constants get wrapped as tensors in AOT Autograd.  So we want this to work for tensors.

Reply:
But we still need to check that it's 0d and that it's dtype is correct for the other inputs (e.g. integral inputs and float alpha error out in eager)